internal/grpc/services/ocminvitemanager/ocminvitemanager.go

```diff
@@ -58,7 +58,7 @@ func (s *service) Close() error {
 }
 
 func (s *service) UnprotectedEndpoints() []string {
-	return []string{}
+	return []string{"/cs3.ocm.invite.v1beta1.InviteAPI/AcceptInvite"}
```

Comment:
You can remove this, since the accept endpoint would be calling other GRPC methods for which we need to provide authentication anyway.